### PR TITLE
SOCIS student

### PR DESCRIPTION
**Pull request by OVSofia**
Please note that if you are trying to update **your** website, this is the wrong place to do so. Please carefully follow the Beautiful Jekyll instructions (found at https://github.com/daattali/beautiful-jekyll#readme) and make sure you submit changes to **your** version of the project.

If your intention is to submit a Pull Request, please describe what your pull request achieves.

Thank you!

### CODE REVIEW

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: SOCIS student
+author: Sofia
```
Comment (shreyasbapat) [inline _posts/2019-05-29-Student-SOCIS-2019.md]: sofia - In small

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: SOCIS student
```
Comment (shreyasbapat) [inline _posts/2019-05-29-Student-SOCIS-2019.md]: A more informative title maybe?

**Comment by shreyasbapat** [comment]
Welcome @OVSofia . Your one of the first contributions in here. I have left two comments.

File: _posts/2019-05-29-Student-SOCIS-2019.md (hunk 2 above)
Comment (OVSofia) [reply _posts/2019-05-29-Student-SOCIS-2019.md]: Some suggestion?

Comment (ritzvik) [reply _posts/2019-05-29-Student-SOCIS-2019.md]: Maybe "Here's to Summer of Code with EPY" ?

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+## Thanks to EinsteinPy and the European Space Agency
+I am so glad on being selected as a "Summer Developer" in ESA's Summer of Code in Space with EinsteinPy.
+
+First of all, I want to express my sincere gratitude to EinsteinPy and to the European Space Agency for the opportunity they gave to me.
+I am sure it is going helpful to develop my skills in computer science. So I hope my collaboration will be useful to EinsteinPy as well.
+
+##  Why EinsteinPy?
+I am truly believe that EinsteinPy project is going to be a powerful tool for researchers working in gravitational physics field.
```
Comment (ritzvik) [inline _posts/2019-05-29-Student-SOCIS-2019.md:19]: I truly*

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+I am so glad on being selected as a "Summer Developer" in ESA's Summer of Code in Space with EinsteinPy.
+
+First of all, I want to express my sincere gratitude to EinsteinPy and to the European Space Agency for the opportunity they gave to me.
+I am sure it is going helpful to develop my skills in computer science. So I hope my collaboration will be useful to EinsteinPy as well.
+
+##  Why EinsteinPy?
+I am truly believe that EinsteinPy project is going to be a powerful tool for researchers working in gravitational physics field. 
+The existence of simulating platforms in languages like C or C++ was a great step. 
+However, these languages are not as easy to handle for someone without a high programming skills. And even though it is usual that theoretical 
+physicist have some experience in computational areas, they shouldn't spend their energy and time in such coding process.
```
Comment (ritzvik) [inline _posts/2019-05-29-Student-SOCIS-2019.md:22]: physicists*

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+First of all, I want to express my sincere gratitude to EinsteinPy and to the European Space Agency for the opportunity they gave to me.
+I am sure it is going helpful to develop my skills in computer science. So I hope my collaboration will be useful to EinsteinPy as well.
+
+##  Why EinsteinPy?
+I am truly believe that EinsteinPy project is going to be a powerful tool for researchers working in gravitational physics field. 
+The existence of simulating platforms in languages like C or C++ was a great step. 
+However, these languages are not as easy to handle for someone without a high programming skills. And even though it is usual that theoretical 
+physicist have some experience in computational areas, they shouldn't spend their energy and time in such coding process. 
+Hence, they can focus all their efforts on their theoretical physics develops.
```
Comment (ritzvik) [inline _posts/2019-05-29-Student-SOCIS-2019.md:23]: development*

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+I am sure it is going helpful to develop my skills in computer science. So I hope my collaboration will be useful to EinsteinPy as well.
+
+##  Why EinsteinPy?
+I am truly believe that EinsteinPy project is going to be a powerful tool for researchers working in gravitational physics field. 
+The existence of simulating platforms in languages like C or C++ was a great step. 
+However, these languages are not as easy to handle for someone without a high programming skills. And even though it is usual that theoretical 
+physicist have some experience in computational areas, they shouldn't spend their energy and time in such coding process. 
+Hence, they can focus all their efforts on their theoretical physics develops.  
+
+That is why python, due to its low learning curve and clean syntax, is becoming by excellence the programming language among the scientific community.
```
Comment (ritzvik) [inline _posts/2019-05-29-Student-SOCIS-2019.md:25]: Could you replace the line with something else. It is not looking grammatically correct.

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+Therefore, that is General Theory of Relativity equations should be included as a python packages.
```
Comment (ritzvik) [inline _posts/2019-05-29-Student-SOCIS-2019.md:26]: package*

File: _posts/2019-05-29-Student-SOCIS-2019.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+The existence of simulating platforms in languages like C or C++ was a great step. 
+However, these languages are not as easy to handle for someone without a high programming skills. And even though it is usual that theoretical 
+physicist have some experience in computational areas, they shouldn't spend their energy and time in such coding process. 
+Hence, they can focus all their efforts on their theoretical physics develops.  
+
+That is why python, due to its low learning curve and clean syntax, is becoming by excellence the programming language among the scientific community.
+Therefore, that is General Theory of Relativity equations should be included as a python packages. 
+
+##  And last but not least... 
+...If you think as I do that science should be an open source project where everyone can contribute, don't be shy and join us!
```
Comment (ritzvik) [inline _posts/2019-05-29-Student-SOCIS-2019.md:29]: "as I do," 
You might want to put a comma there

also ... not required

**Review by ritzvik — CHANGES_REQUESTED** [review]

**Comment by ritzvik** [comment]
Tip for future: Try to make new branch for each PR. Never make changes directly to the master

**Review by Masquerade0097 — COMMENTED** [review]
@OVSofia Thanks for your contribution !! The blog seems good :) 

In addition to @ritzvik 's comment, here are some good resources to learn to work with branches in git [this](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches) & [this](https://www.atlassian.com/git/tutorials/using-branches).

**Pull request merged**